src/enterprise.md

```diff
@@ -2,70 +2,88 @@
 layout: enterprise
 title: Enterprise
 permalink: /enterprise
-custom_css: https://assets.calendly.com/assets/external/widget.css
+has_calendly: true
 ---
 
-# Tilt Enterprise
+<h2 class="Enterprise-heroTitle">Enterprise Ready-Steady-Go</h2>
+<section class="Enterprise-hero">
+  <div class="Enterprise-hero-text">
+    <p class="Enterprise-hero-text-subhead">
+      Our focus is on teams. We’re obsessed about creating stellar dev experiences that keep your engineers productive while working with complex infrastructure.
+    </p>
+    <div class="Enterprise-hero-text-detail">
+      <p>If you’re navigating a complex migration, we can support you with onboarding support, workshops, and dev pairing on-call. Scale with unlimited Snapshots and advanced metrics. We can prioritize your feature requests and give you insights into our roadmap. Need an on-prem version? We’ve got you covered.</p>
+      {% svg assets/svg/illustration-enterprise.svg %}
+    </div>
+  </div>
+  <div class="Enterprise-hero-cta">
+    <h4 class="Enterprise-hero-cta-title">Schedule a Demo</h4>
+    <div class="calendly-inline-widget" data-url="https://calendly.com/dbentley/30min?hide_event_type_details=1"></div>
+  </div>
+</section>
+
+<h3 class="Enterprise-sectionHeading">Team Features</h3>
+<ul class="Enterprise-featureList">
+  <li>
+    <h4 class="Enterprise-featureItem-title">Orderly Orchestration</h4>
+    <p class="Enterprise-featureItem-text">Our engine starts the whole app runs automated rebuilds as you edit in your IDE. Get a continuous feedback loop with your logs, broken builds, runtime errors. </p>
+  </li>
+  <li>
+    <h4 class="Enterprise-featureItem-title">Built-In Best Practices</h4>
+    <p class="Enterprise-featureItem-text">
+      We’ve codified best practices to give your team a common development path and ensure reproducibility. Anyone can start the app – new hires just `tilt up`.
+    </p>
+  </li>
+  <li>
+    <h4 class="Enterprise-featureItem-title">Painless Onboarding</h4>
+    <p class="Enterprise-featureItem-text">
+      We made Tilt platform agnostic, versatile and easy to configure, because we know every setup is different. You can integrate Tilt in stages for a smooth transition.
+    </p>
+  </li>
+</ul>
+
+{% include index2_cta.html %}
+
+<h3 class="Home2-sectionHeading">Learn More</h3>
```

Comment:
this seems like a weird thing to put on the enterprise page.

is this duplicated from the homepage? should it be an include?

Reply:
I agree it's a little odd. It makes a lot of sense visually as it makes the layout feel more complete. But logically it's not a great part of the sell, just yet. 

I think the copy on here will be updated to be more Enterprise-specific, anyhow. So I'm going to leave this section as is for now.